tests/rptest/services/admin.py

```diff
@@ -34,7 +34,25 @@ def should_strip_auth(self, old_url, new_url):
         return False
 
 
+class Replica:
+    node_id: int
+    core: int
+
+    def __init__(self, replica_dict):
+        self.node_id = replica_dict["node_id"]
+        self.core = replica_dict["core"]
+
+    def __getitem__(self, item_type):
+        if item_type == "node_id":
+            return self.node_id
+        elif item_type == "core":
+            return self.core
+        return None
```

Comment:
nit: if we use `.property` syntax instead of  `["property"]` we don't need to define it

Reply:
I want to use this class in existing functions